configure.ac

```diff
@@ -829,7 +829,7 @@ export MPLLIBNAME
 # with-device
 if test "$with_device" = "default" ; then
     # Pick the device.  For now, always choose ch3
-    with_device=ch3
+    with_device=ch4
```

Comment:
When users do `./configure`, they will hit the `ch4` error message directly. This can be puzzling especially when the user is trying to do ch3 build. I suggest do a `AC_MSG_ERROR` here as well but with a more complete help message -- in addition to the following ch4 error message, also add `To build ch3, use --with-device=ch3`.